src/proof/alethe/alethe_post_processor.cpp

```diff
@@ -1539,30 +1539,78 @@ bool AletheProofPostprocessCallback::update(Node res,
     }
     // ======== Alpha Equivalence
     //
-    // Given the formula F = (forall ((y1 A1) ... (yn An)) G) and the
-    // substitution sigma = {y1 -> z1, ..., yn -> zn}, the step is represented
+    // Given the formula F := (forall ((y1 A1) ... (yn An)) G) and the
+    // substitution sigma is {y1 -> z1, ..., yn -> zn}, the step is represented
     // as
     //
     //  ------------------ refl
     //  (cl (= G G*sigma))
     // -------------------- bind, z1 ... zn (= y1 z1) ... (= yn zn)
-    //  (= F F*sigma)
+    //  (cl (= F (forall ((z1 A1) ... (zn An)) G*sigma)))
     //
-    // In case the sigma is the identity this step is merely converted to
+    // unless sigma is the identity, in which case this step is converted to
     //
     //  ------------------ refl
     //  (cl (= F F))
+    //
+    // When F := (forall ((y1 A1) ... (yk Ak)) G) and sigma is the substitution
```

Comment:
The structure is a little confusing. Right now it seems that you say:

The general form is this. The result is always as follows.
Unless Case B (sigma is id) applies, then result is as follows.

Another form is this. It represents a case where XY. Then, the result is as follows.

Reply:
I've reorganized the documentation so that it first lists the special cases then the unrestricted case. PTAL